.github/workflows/maven-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release to Maven Staging
+
+on:
+  push:
+    branches:
+      - master
+  # temporary to test it's working
+  pull_request:
+
+jobs:
+  build:
+    name: Build project and publish release
```

Comment:
So this now publishes both -SNAPSHOT and release, maybe update to say that?

Reply:
Replaced by deploy which is what is really happening.